packages/components/src/color-picker/README.md

```diff
@@ -1,24 +1,57 @@
 # ColorPicker
 
-Accessible color picker.
+<div class="callout callout-alert">
```

Comment:
Since the previous `ColorPicker` was not marked experimental, I wonder if we should remove this paragraph